---
layout: page
title: Software
description: List of software developed by our group
---
**In alphabetical order ...**

### For single cell data analysis:

**Alleloscope** (allele-specific copy number estimation for scDNA and scATAC sequencing)
([Github](https://github.com/seasoncloud/Alleloscope))

**cTP-Net** (single cell Transcriptome to Protein prediction with deep neural network)
([Github](https://github.com/zhouzilu/cTPnet/))

**DENDRO** (Genetic heterogeneity profiling by scRNA-seq)
([Github](https://github.com/zhouzilu/DENDRO))

**DESCEND** (Expression distribution deconvolution for single cell RNA-seq)
([Github](https://github.com/jingshuw/descend))

**MUSIC** (Bulk expression deconvolution with scRNA-seq reference) 
([Github](https://github.com/xuranw/MuSiC))

**SAVER** (Gene expression imputation and denoising for single cell RNA sequencing)
([Github](https://github.com/mohuangx/SAVER))

**SAVER-X** (SAVER harnessing external data)
([Github](https://singlecell.wharton.upenn.edu/saver-x/))

**SCALE** (Single cell allele-specific expression analysis)  
([Github](https://github.com/yuchaojiang/SCALE))

**TASC** (Toolkit for noise modeling in single cell RNA-seq with spike-ins)]
([Github](https://github.com/scrna-seq/TASC)]

<br/><br/> 

### For copy number intra-tumor heterogeneity analysis:

**CANOPY** (Tumor phylogeny reconstruction by spatial and temporal bulk RNA sequencing)
([CRAN](https://cran.r-project.org/web/packages/Canopy/))

**FALCON** (Allele-specific copy number estimation using whole genome sequencing data)
([CRAN](https://cran.r-project.org/web/packages/falcon/index.html))

**FALCON-X** (Allele-specific copy number estimation using whole exome sequencing data)
([CRAN](https://cran.r-project.org/web/packages/falconx/index.html))

**CODEX/CODEX2** (statistical framework for full-spectrum CNV profiling in whole genome, whole exome, and targeted DNA sequencing) 
([Github](https://github.com/yuchaojiang/CODEX2))

**iCNV** (Integration across array and sequencing platforms for copy number detection)
([Github](https://github.com/zhouzilu/iCNV))

**MARATHON** (Comprehensive pipeline for copy number profiling in normal and tumor samples)
([Github](https://github.com/yuchaojiang/MARATHON))

**SWAN** (Structural variant profiling using paired-end genome sequencing data)
([Bitbucket](https://bitbucket.org/charade/swan/overview))


<br/><br/> 

### General statistical tools:

**GCAT** (Two-sample tests for categorical data utilizing similarity information among the categories)
([CRAN](https://cran.r-project.org/web/packages/gCat/index.html))

**GSEG** (Change-point detection for multivariate data through a similarity graph on the observations)
([CRAN](https://cran.r-project.org/web/packages/gSeg/index.html))

**LEAPP** (Latent factor (“batch effect”) adjustment in multiple hypothesis testing)
([CRAN](https://cran.r-project.org/web/packages/leapp/index.html))

**SEMBLANCE** (rank-semblance kernel for data compression, niche detection, and feature extraction)
([CRAN](https://cran.r-project.org/web/packages/Semblance/index.html))

**SEQCBS** (Segmentation and Bayesian confidence interval calculation for matched case/control point processes)
([CRAN](https://cran.r-project.org/web/packages/seqCBS/index.html))

