---
layout: page
title: Software
description: List of software developed by our group
---

Most of my software as well as vignettes and scripts for the analyses connnected to my research can be found on my [GitHub page](https://github.com/jinzhuangdou). I maintain the following software packages:


* [Monopogen](https://github.com/KChen-lab/Monopogen) (Single nucleotide variant calling from single cell sequencing) 

Monopogen is developed to achieve SNV calling from single cell RNA sequencing, single cell ATAC sequencing.

* [BindSC](https://github.com/KChen-lab/bindSC) (Bi-order integration of single cell multi-omics data) 

This tool is developed to achive bi-order integration (in silico multi-omics data) of single cell RNA sequencing, single cell ATAC sequencing, spacial transcriptomics and CyTOF data. 

* [WEScall](https://github.com/dwuab/WEScall) (A genotype calling pipeline for whole exome sequencing (WES) data)

WESCall can analyze both target and off-target data, and is a useful tool to facilitate WES sutides with decent amount of off-target data.

* [SEEKIN](https://github.com/chaolongwang/SEEKIN)(Sequencing-based estimation of kniship and inbreeding coefficients)

This program estimates the kinship and inbreeding coefficients based on extremely low-sequencing coverage datasets (typically lower than 0.5X). 

* [AMMO](https://github.com/jinzhuangdou/AMMO)(An integrated package for whole-genome restriction mapping and genome scaffolding)

This is an integrated program that generates 2b-RAD-based restriction maps for scaffolding draft genome assemblies produced by using short Illumina reads or long PacBio reads. 

* [RADTyping](https://github.com/jinzhuangdou/RADtyping)(De novo RAD Genotyping in Mapping Populations)

This is an integrated package for accurate De Novo codominant and dominant RAD genotyping in mapping populations. 

<br/><br/> 

